ios/FluentUI.Demo/FluentUI.Demo/Demos/ButtonDemoController.swift

```diff
@@ -181,30 +181,30 @@ extension ButtonDemoController: DemoAppearanceDelegate {
 
     private var themeWideOverrideButtonTokens: [ButtonTokenSet.Tokens: ControlTokenValue] {
         return [
-            .titleFont: .fontInfo { FontInfo(name: "Times", size: 20.0, weight: .regular) },
-            .backgroundColor: .dynamicColor {
-                return DynamicColor(light: GlobalTokens.sharedColors(.marigold, .shade30),
-                                    dark: GlobalTokens.sharedColors(.marigold, .tint40))
+            .titleFont: .font { UIFont(name: "Times", size: 20.0)! },
```

Comment:
Do we need the force unwrap? Could we either switch to a `guard let` or use this [init](https://developer.apple.com/documentation/uikit/uifont/1619025-init) instead?

Reply:
fixed